client/ayon_core/plugins/publish/collect_aovs_to_anatomy_instance_data.py

```diff
@@ -0,0 +1,22 @@
+import pyblish.api
+
+
+class CollectAOVNameToAnatomyInstanceData(pyblish.api.InstancePlugin):
+    """
+    Collect AOV name to template data.
+
+    Note:
+        This functionality depends on AOV name added by
+        ``submit_publish_job`` plugin and so it only works
+        with Deadline submission.
```

Comment:
If so, shouldn't this be inside the `Deadline` addon to begin with?

Reply:
It is stated there, but in essence this can also be utilized inside RoyalRender if needed.